### PR TITLE
replaced clock and keyboard

### PR DESCRIPTION
**Pull request by IanKBovard**
replaced clock and keyboard

### CODE REVIEW

**Comment by IanKBovard** [comment]
clock
new
Screen Shot 2020-07-15 at 3 33 39 PM
old
Screen Shot 2020-07-16 at 11 01 41 AM

keyboard
new
Screen Shot 2020-07-15 at 3 31 27 PM

old
Screen Shot 2020-07-16 at 11 02 48 AM

File: src/js/__tests__/__snapshots__/metadata-test.js.snap [hunk 1]
```diff
@@ -1898,8 +1896,8 @@ Object {
     "reactjs",
   ],
   "Keyboard": Array [
-    "accessibility",
-    "tab",
+    "keyboard",
```
Comment (ShimiSun) [inline src/js/__tests__/__snapshots__/metadata-test.js.snap:1899]: Why was 'accessibility' and 'tab' got removed? we should keep them in addition to the new keywords.

File: src/js/__tests__/__snapshots__/metadata-test.js.snap [hunk 2]
```diff
@@ -540,11 +540,9 @@ Object {
     "report",
   ],
   "Clock": Array [
-    "date",
-    "timer",
-    "hands",
-    "hour",
-    "watch",
+    "clock",
```
Comment (ShimiSun) [inline src/js/__tests__/__snapshots__/metadata-test.js.snap:543]: we should keep the previous list along with the new keywords.

File: src/js/metadata.js [hunk 3]
```diff
@@ -85,7 +85,7 @@ export default {
   'ClearOption': ['paintbrush'],
   'Cli': ['dos', 'line', 'command', 'console'],
   'Clipboard': ['document', 'list', 'report'],
-  'Clock': ['date', 'timer', 'hands', 'hour', 'watch'],
+  'Clock': ['clock', 'core', 'date', 'hands', 'hour', 'time', 'timer', 'watch'],
```
Comment (ShimiSun) [inline src/js/metadata.js:88]:
```suggestion
  'Clock': ['core', 'date', 'hands', 'hour', 'time', 'timer', 'watch'],
```
The name of the icon is used as default search option, so it doesn't need to be added to the metadata.

File: src/js/metadata.js [hunk 4]
```diff
@@ -293,7 +293,7 @@ export default {
   'Iteration': ['cascade', 'iterate', 'repeat', 'stack', 'steps'],
   'Java': ['coffee', 'code', 'coding', 'cup'],
   'Js': ['javascript', 'logo', 'react', 'reactjs'],
-  'Keyboard': ['accessibility', 'tab'],
+  'Keyboard': ['accessibility', 'keyboard', 'tab', 'technology'],
```
Comment (ShimiSun) [inline src/js/metadata.js:296]:
```suggestion
  'Keyboard': ['accessibility', 'tab', 'technology'],
```
The name of the icon is used as a default search option, so it doesn't need to be added to the metadata.

**Pull request merged**